src/server/js/resources/houses/schema.js

```diff
@@ -0,0 +1,17 @@
+const typeDefs = `
+    type Address {
+        street: String
+        city: String
+        state: String
+        zip: String
+    }
+
+    type House {
+        id: ID!
+        address: Address
+        administrators: [User]
+        name: String
+    }
+`;
+
+export { typeDefs };
```

Comment:
in each model's schema file, there are two parts:
- the schema definition, ie `typeDefs`, in which the type for `XyzModel` is defined as `Xyz` (note that for nested docs, you also need to define the type)
- the resolvers for `Xyz`'s fields, which is optional if all its fields resolve to scalars or other defined types

both of these are exported as named exports to be used in root schema file (see below)